examples/grpc-client-streaming/grpc_client_streaming_client.out

```diff
@@ -1,11 +1,11 @@
-# To run the client and send request, create ballerina project and package
-# inside the project and include below files to package directory. Build the
-# project using build-in ballerina tool.
+# Create a Ballerina project and a package inside it. 
+# Add the following files inside the package directory. Build the
+# project using the built-in Ballerina tool.
```

Comment:
The following needs to be rephrased, if this built-in tool referred to here is the build command: "Build the project using the built-in Ballerina tool."

Reply:
@shavidissa, shall we make it like "Add the following files inside the package directory and build the project".
Thanks

Reply:
Please close without merging. Fix can be found in https://github.com/ballerina-platform/ballerina-examples/pull/777